### ISSUE TITLE
Report shows "Emulated Moto G4" when testing on a mobile device

### ISSUE THREAD

**Issue by ericandrewlewis** [GoogleChrome/lighthouse#14560]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

Can not share

### What happened?

I followed the section on [Testing on a mobile device](https://github.com/GoogleChrome/lighthouse/blob/main/docs/readme.md#testing-on-a-mobile-device) to run Lighthouse on a URL on a Moto Power G.

The exact command I ran was

```
lighthouse \
  --port=9222 \
  --screenEmulation.disabled \
  --throttling-method=devtools \
  --throttling.cpuSlowdownMultiplier=1 \
  --no-emulatedUserAgent \
  --view \
  --print-config \
  SOME_URL
```

I see Lighthouse load the page in Chrome on the phone. However, in the report it says "Emulated Moto G4 with Lighthouse 9.6.8":

![Screen Shot 2022-11-23 at 1 54 53 PM](https://user-images.githubusercontent.com/1087646/203626047-8d6ae837-7a98-4991-84f4-592e2a274802.png)

### What did you expect?

The LH page would say "Testing on a real device with Lighthouse 9.6.8", or even "Testing on physical Moto Power G device with Lighthouse 9.6.8"

### How were you running Lighthouse?

CLI

### Lighthouse Version

9.6.8

### Chrome Version

107

### Node Version

16.13.2

### OS

Mac OS

**Comment by ericandrewlewis** [comment]
[This](https://github.com/GoogleChrome/lighthouse/blob/main/report/renderer/report-renderer.js#L147) is where the `Emulated Moto G4 with Lighthouse 9.6.8` string is built.

That depends on the [`deviceEmulation` value created here](https://github.com/GoogleChrome/lighthouse/blob/e3f2337d73b7b9e6a9ff01b85cd0750e8e10e86a/report/renderer/util.js#L515-L518).

I don't believe`settings.formFactor` can be null, it must either be `mobile` or `desktop`, at least when invoking the option with the CLI. So seems like a bug?

**Comment by adamraine** [comment]
I found a couple confusing things going on here:
- `settings.formFactor` represents how the report is scored, not what device is being used. However the report seems to indicate that `settings.formFactor` reflects the emulated device.
- The string will say "Emulated Moto G4" for *any* settings with mobile form factor, even if a different mobile device is being emulated or a real device is being used.

We could use `settings.screenEmulation` to determine this string and then generalize the mobile version so it's not specific to the Moto G4. Ex "Emulated Mobile Device with Lighthouse 9.6.8."

Another option would be to reword the strings like "Score based on Moto G4 with Lighthouse 9.6.8" and leave the `settings.formFactor` logic in place.

**Comment by benschwarz** [comment]
This was discussed previously (and @paulirish explored it in #11796). 

I'd opt to leave `formFactor` as is and update the mobile descriptor to be more generic (i.e.: Not mention a specific device, because people get hung up on that anyway).

**Comment by paulirish** [comment]
@benschwarz do ya'll even use the report renderer? this change doesn't affect the LHR.. just how its interpretted in the report renderer. 
anyway.. draft fix is up in #14587

**Comment by benschwarz** [comment]
> @benschwarz do ya'll even use the report renderer? 
Ah. My bad. Nope, we don't use the report renderer 🙇‍♂️
